astro/src/content/docs/apis/users.mdx

```diff
@@ -336,7 +336,7 @@ The response for this API contains the information for the User that was reactiv
 
 ## Import Users
```

Comment:
I would add a note here: https://fusionauth.io/docs/lifecycle/manage-users/search/search#reindexing-elasticsearch about reindexing after a user import too:

"If you stand up FusionAuth with a database dump and restore, you may need to run this operation" ->

"If you stand up FusionAuth with a database dump and restore or import users using the [User Import API](/docs/apis/users#import-users), you should to run this operation"